DeveloperGame/DeveloperGame/ComputerGeekGame.cs

```diff
@@ -148,5 +200,97 @@ private void PrintLeaderBoard()
                 Console.WriteLine($"Name: {playerScore.Name} | Point(s): {playerScore.Points} | Time: {playerScore.Time}\n");
             }
         }
+
+        private void RandomizeMultipleChoiceQuestions()
+        {
+            List<MultipleChoiceQuestion> tempQuestionList = new List<MultipleChoiceQuestion>();
+            Random random = new Random();
+
+            tempQuestionList.AddRange(_multipleChoiceQuestions);
+            _multipleChoiceQuestions.Clear();
+
+            while (tempQuestionList.Count > 0)
+            {
+                int randomIndex = random.Next(0, tempQuestionList.Count - 1);
+
+                _multipleChoiceQuestions.Add(tempQuestionList[randomIndex]);
+                tempQuestionList.Remove(tempQuestionList[randomIndex]);
+            }            
+        }
+
+        private void RandomizeTrueOrFalseQuestions()
+        {
+            List<TrueOrFalseQuestion> tempQuestionList = new List<TrueOrFalseQuestion>();
+            Random random = new Random();
+
+            tempQuestionList.AddRange(_trueOrFalseQuestions);
+            _trueOrFalseQuestions.Clear();
+
+            while (tempQuestionList.Count > 0)
+            {
+                int randomIndex = random.Next(0, tempQuestionList.Count - 1);
+
+                _trueOrFalseQuestions.Add(tempQuestionList[randomIndex]);
+                tempQuestionList.Remove(tempQuestionList[randomIndex]);
+            }            
+        }
+
+        private MultipleChoiceQuestion.Answers GetMcAnswerFromPlayer()
+        {               
+
+            while (true)
+            {
+                Console.Write("\n\tAnswer: ");
+                string answerStg = Console.ReadLine();
+
+                if(answerStg.Length < 1 || string.IsNullOrWhiteSpace(answerStg))
```

Comment:
This whole while loop is very off, you only tell them invalid input every other answer, it should be everytime